content/300-accelerate/250-connection-pooling.mdx

```diff
@@ -4,39 +4,45 @@ metaTitle: 'Accelerate: Connection Pooling'
+To learn more about connection pools and connection pooling, be sure to check out [Prisma's Data Guide](https://www.prisma.io/dataguide/database-tools/connection-pooling) for a comprehensive overview on what connection pooling is, what it seeks to address, and how it works.
 
-The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable. 
+:::
 
-Accelerate provides built-in connection pooling by default. By simply using Accelerate, you get the benefits of connection pooling without having to configure anything. However, you can also configure the connection pool to suit your needs.
+:::note
 
-<Admonition>
+For more information about the internal connection pool of Prisma ORM specifically, see our [ORM connection pooling documentation](/orm/prisma-client/setup-and-configuration/databases-connections/connection-pool).
 
-For more information about connection pooling in Prisma, see the documentation [here](/orm/prisma-client/setup-and-configuration/databases-connections/connection-pool).
+:::
 
-</Admonition>
+Accelerate provides built-in connection pooling by default. By using Accelerate, you get the benefits of connection pooling without having to configure anything. The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable.
+
+In some cases, however, it may be beneficial to further configure connection pooling in order to optimize the performance of your application.
 
-### Default connection pool size
+### Connection Pooling in Accelerate
 
-Accelerate has a default `connection_limit` set to `10`.
+By default, Accelerate has a default connection limit of `10`. In most cases, this connection limit is sufficient. However, in high traffic or compute environments, you may need to [increase this value](#configuring-the-connection-pool-size). For the [Pro plan]() there is a maximum connection limit of `100`. For [Business plan]() users the maximum is `1000`.
 
 ### Configuring the connection pool size
 
-The connection pool size can be configured via the _database connection string_ in your Platform project (_not_ the Accelerate connection string in your Prisma schema) via the `connection_limit` parameter. For example, set a connection pool size of `10` like this: `postgresql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=10`.
+The connection pool size can be configured in your Platform project via the `Connection limit` setting. For example, set a connection pool size of `10` like this:
+
+![Update database connection string in Accelerate](/img/accelerate/accelerate-update-database-connection-string.png)
```

Comment:
@luanvdw this is a filler image, we'll need this.

Reply:
Can we not put it in the URL anymore?

Reply:
Correct. @rtbenfield let me know if I'm wrong here.

Reply:
No, we disregard the value from the URL when the new explicit setting is active. This was needed for backwards compatibility with existing users that may have a higher value in the URL than their plan allows. Accelerate will continue to work for them as it does today until they update their connection string and are forced to choose from the new options.

Reply:
hmmm, we should probably note this? It does seem odd though. So essentially I wont be able to set the connection limit in the URL if I'm a new users. But if I'm old and I have it set it will still work but be limited to my plan regardless of what I put in the string?

Reply:
> But if I'm old and I have it set it will still work but be limited to my plan regardless of what I put in the string?
The plan limits are also not enforced for existing users until they update their Accelerate config. That's also intention to preserve the behavior that exists today. We don't want any users to have a sudden impact from the change and rather want to give them the opportunity to adopt the new behavior on their own terms.

Reply:
> The plan limits are also not enforced for existing users until they update their Accelerate config.

What does this mean? Is this simply toggling on the setting in the workspace UI?

Reply:
Right. @luanvdw maybe you have some thoughts on how that flow is in the Console? Any designs we can share here?